tests/Resources/Page/ODH/AiApps/Pachyderm.robot

```diff
@@ -6,7 +6,10 @@ Library             SeleniumLibrary
 *** Keywords ***
 Uninstall Pachyderm Operator
     [Documentation]    Uninstall pachyderm operator and its related component.
-    Go To    ${OCP_CONSOLE_URL}
+    Run Cell And Check For Errors   !pachctl delete pipeline edges
+    Close All Browsers
+    Open Browser  ${OCP_CONSOLE_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+    LoginPage.Login To Openshift  ${OCP_ADMIN_USER.USERNAME}  ${OCP_ADMIN_USER.PASSWORD}  ${OCP_ADMIN_USER.AUTH_TYPE}
```

Comment:
I think you can remove lines 10-11-12 from here. You need to add the Teardown for jupyter testing (check other suites in ods-ci), maybe you could compose a custom teardown kw

Reply:
I have added one more keyword which cleans and stops the server.